Realm/RLMObjectStore.mm

```diff
@@ -216,7 +216,18 @@ static bool RLMRealmCreateTables(RLMRealm *realm, RLMSchema *targetSchema, bool
         }
     }
 
-    // FIXME - remove deleted tables
+    if (updateExisting) {
+        NSMutableArray *tablesToDelete = [NSMutableArray array];
```

Comment:
> You should use an NSMutableSet and store the RLMStringData instead of the NSString to avoid doing another conversion further down.

Won't an `NSSet` not hold `tightdb::StringData` since it's not an Objective-C object?